### DISCUSSION TITLE
Warning flag on dApp

### DISCUSSION THREAD

**Discussion by gabrielhicks** [phantom/docs#76] [Help & Support] [answered]
### Summary

When visiting a dApp deployed to a new subdomain, an error is prompted to users when interacting.

### Example

https://rewards.raposacoffee.com/

### Steps to Reproduce

Visit site, interact with anything. Currently you would need to be an active participant in this community to be able to interact.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello,

This should now be resolved. Thank you!

**Comment by gabrielhicks** [comment]
🥇 Thank you! 

We also recently deployed https://radiant.nexus/ and just have a missing review yellow flag on that!

**Reply by adamdelphantom** [reply]
I believe this has since been resolved. Please let me know if not.
